app/src/main/java/org/schabi/newpipe/local/dialog/PlaylistAppendDialog.java

```diff
@@ -26,8 +26,12 @@
 public final class PlaylistAppendDialog extends PlaylistDialog {
     private static final String TAG = PlaylistAppendDialog.class.getCanonicalName();
 
+    private static final float DEFAULT_ALPHA = 1f;
+    private static final float GRAYED_OUT_ALPHA = 0.3f;
```

Comment:
0.6 would also be fine I think. Greying out too much could make text not readable anymore.